andela/dark-knight-cfh
# #157253519 Fix signup with social media functionality

#### What does this PR do?
Fix signup with social media functionality 

#### Description of Task to be completed?
1. Discard signup with github implementations
2. Cleanup codebase for signup functionality
3. Include token in response after signup using social media
4. Save token to localstorage upon signup using social media

#### How should this be manually tested?
On console: run command
1. `npm start`
2. On browser, on app landing page, click signup button
3. Click twitter, facebook or google icons, which will take you to the social site
4.  Login from social media to be redirected to the app
5. Check browser applications from developer tools to confirm that signup token exists

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#157253519](https://www.pivotaltracker.com/story/show/157253519)

#### Screenshots (if appropriate)
N/A
